jest.config.js

```diff
@@ -0,0 +1,15 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
```

Comment:
Use `testMatch` to detect test files. Here we should ignore `./tests/**`, while `./tests/jest.config.js` should include only them